monkey/monkey_island/cc/ui/src/components/pages/ConfigurePage.js

```diff
@@ -175,6 +163,26 @@ class ConfigurePageComponent extends AuthComponent {
     fileDownload(JSON.stringify(this.state.configuration, null, 2), 'monkey.conf');
   };
 
+  sendConfig() {
+    return (
+      this.authFetch('/api/configuration/island',
+      {
+        method: 'POST',
+        headers: {'Content-Type': 'application/json'},
+        body: JSON.stringify(this.state.configuration)
+      })
+      .then(res => {
+        if (!res.ok)
+        {
+          throw Error()
+        }
+        return res;
+      }).catch(error => {
+        console.log('bad configuration');
```

Comment:
`console.warn` instead()?

Reply:
I don't think this would solve anything. I think this statement could be removed altogether, but it's not my code so I tried not to change it.